exercises/react/src/example/kotlin/React.kt

```diff
@@ -0,0 +1,40 @@
+class Reactor<T>() {
+    abstract inner class Cell {
+        abstract val value: T
+        internal var dependents = mutableListOf<ComputeCell>()
```

Comment:
interestingly, `protected` is not possible, because then the `c.dependents.add(this)` in the constructors all fail compilation with:

```
:compileKotlin
e: /home/pt/xkotlin/exercises/react/src/main/kotlin/React.kt: (22, 15): Cannot access 'dependents': it is protected in 'Cell'
e: /home/pt/xkotlin/exercises/react/src/main/kotlin/React.kt: (25, 16): Cannot access 'dependents': it is protected in 'Cell'
e: /home/pt/xkotlin/exercises/react/src/main/kotlin/React.kt: (26, 16): Cannot access 'dependents': it is protected in 'Cell'
 FAILED
```

This is a little strange to me, since `ComputeCell` is (I hope?!) a subclass of `Cell`. Perhaps it doesn't apply inside constructors, or something like that...